packages/base/src/Boot.js

```diff
@@ -40,6 +41,7 @@ const boot = async () => {
 			F6Navigation.init();
 		}
 
+		attachStylesToHead();
```

Comment:
IMO boot.js should not have knowledge of this functionality, if possible.

Instead, I would suggest to hook this `attachStylesToHead` code to the `applyTheme` function, which is called a few lines below. If you do this, this will also automatically trigger when calling `setTheme`, as it calls `applyTheme` internally, so you won't need to change that function either.